site/content/en/docs/Tasks/loadbalancer.md

```diff
@@ -14,7 +14,7 @@ A LoadBalancer service is the standard way to expose a service to the internet.
 
 ## Using `minikube tunnel`
 
-Services of type `LoadBalancer` can be exposed via the `minikube tunnel` command. It will run in a separate terminal until Ctrl-C is hit.
+Services of type `LoadBalancer` can be exposed via the `minikube tunnel` command. It will run in the terminal until Ctrl-C is hit.
```

Comment:
I agree the wording before wasn't clear. But I'm not sure we are properly explaining now as well, the idea here is that the user must leave the tunnel running in a different terminal to have the LoadBalancer services exposed. WDYT?

Reply:
I think word separate was meant to make sure user know they need two terminals for this.

Reply:
Better?